services/pull/merge.go

```diff
@@ -62,6 +62,8 @@ func GetDefaultMergeMessage(ctx context.Context, baseGitRepo *git.Repository, pr
 		if err != nil {
 			if !git.IsErrNotExist(err) {
 				return "", "", err
+			} else if mergeStyle == repo_model.MergeStyleRebase {
+				return "", "", nil
```

Comment:
Now `err != nil` and `!git.IsErrNotExist(err)`, but it returns `err = nil`?

Reply:
I don't remember why I added this and it seems unnecessary, removed.